lib/reek/smells/attribute.rb

```diff
@@ -1,5 +1,5 @@
 require_relative 'smell_detector'
-require_relative '../core/smell_configuration'
+require_relative '../smells/smell_configuration'
```

Comment:
You can now remove the `../smells/` bit from the require here and in other files under `smells/`.